modules/controller/world_model_tracking.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
```

Comment:
Nit: Add docstring at start of file. Can be short, does not have to be anything fancy.